modules/collect/src/main/java/com/opengamma/strata/collect/Guavate.java

```diff
@@ -772,6 +773,63 @@ public R next() {
     return collectingAndThen(toImmutableList(), Guavate::combineFuturesAsList);
   }
 
+  //-------------------------------------------------------------------------
+  /**
+   * Converts a map of futures to a single future.
+   * <p>
+   * This is similar to {@link #combineFuturesAsList(List)} but for maps.
+   * Effectively, this converts {@code Map<K, CompletableFuture<T>>} to {@code CompletableFuture<Map<K, T>>}.
```

Comment:
Probably worth using the same type param names in the Javadoc and the method signature (`K` and `T` vs `K` and `V`)